src/main.ts

```diff
@@ -16,16 +17,23 @@ export async function run() {
     // Execute bridge command
     await sb.executeBridgeCommand(formattedCommand, getWorkSpaceDirectory())
   } catch (error) {
+    info('#catch errorss')
     throw error
   } finally {
     await cleanupTempDir(tempDir)
+    info('Synopsys Action Workflow Execution Completed')
```

Comment:
we shouldn't use camel case inbetween sentence Synopsys Action is fine but workflow execution completed should be in small case.

Reply:
fixed